nemo_text_processing/text_normalization/en/data/roman/regnal_to_spoken.tsv

```diff
@@ -0,0 +1,18 @@
+II	the two
+III	the three
+IV	the four
+V	the five
+VI	the six
+VII	the seven
+VIII	the eight
+IX	the nine
+X	the ten
+XI	the eleven
+XII	the twelve
+XIII	the thirteen
+XIV	the fourteen
+XV	the fifteen
+XVI	the sixteen
+XVII	the seventeen
+XVIII	the eighteen
+XIX	the nineteen
```

Comment:
Curious why we only need to specify this short list?

Reply:
The original filter for names only selected from index 2=>19. I couldn't find a "spec" per say, but given how Regnal numbers work, I guess it's unlikely you'll see anything beyond Julius Caesar XIX+ so reduce the FAR output where ever possible I guess? That's my working theory.